src/common/Spinner.tsx

```diff
@@ -30,8 +31,19 @@ export interface SpinnerProps extends Omit<BoxProps, "width" | "height" | "borde
  * // Custom size and colors
  * <Spinner size="6" borderColor="blue.500" />
  */
-export const Spinner = ({ size, ...props }: SpinnerProps) => (
+export const Spinner = ({ size, className, ...props }: SpinnerProps) => (
   // Creates a circular spinning animation using border properties
   // The transparent top border creates the illusion of a rotating segment
-  <Box width={size} height={size} animation="spin" borderRadius="full" border="2" borderStyle="solid" borderColor="white" borderTopColor="transparent" {...props} />
+  <Box
+    width="5"
+    height="5"
+    className={`${icon({ size })} ${className}`}
```

Comment:
Didn't we say we were going to rename this?

Reply:
What should we rename to :) maybe square({ size: xx })

Reply:
Yes. That works